spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/iceberg/IcebergPropertiesConstants.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.spark.connector.iceberg;
+
+import com.datastrato.gravitino.catalog.lakehouse.iceberg.IcebergCatalogPropertiesMetadata;
+import com.google.common.annotations.VisibleForTesting;
+
+public class IcebergPropertiesConstants {
+
+  @VisibleForTesting
+  public static final String LAKEHOUSE_ICEBERG_CATALOG_BACKEND =
```

Comment:
could you change `LAKEHOUSE_ICEBERG` to `GRAVITINO_ICEBERG` to keep consistent with `HivePropertiesConstants`?

Reply:
updated.